### PR TITLE
Add datatype converters for various MD filetypes

### PR DESCRIPTION
**Pull request by simonbray**

### CODE REVIEW

**Comment by simonbray** [comment]
I guess I should make the test data a bit smaller, will work on that.

File: lib/galaxy/datatypes/converters/xtc_to_trr.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        ln -s '$input1' ./input.xtc &&
+        mdconvert ./input.xtc -o ./output.trr 
+    ]]></command>
+    <inputs>
+        <param format="xtc" name="input1" type="data" label="Choose XTC"/>
+    </inputs>
+    <outputs>
+        <data format="trr" name="output1" from_work_dir="output.trr"/>
```
Comment (mvdbeek) [inline lib/galaxy/datatypes/converters/xtc_to_trr.xml]: You can combine the converters that convert to the same datatype in a single tool by allowing more input dataset formats.

File: lib/galaxy/datatypes/converters/xtc_or_trr_to_dcd.xml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    <inputs>
+        <param format="xtc,trr" name="input1" type="data" label="Choose input file"/>
+    </inputs>
+    <outputs>
+        <data format="dcd" name="output1" from_work_dir="output.dcd"/>
+    </outputs>
+    <tests>
+        <test>
+            <param name="input1" format="xtc" value="traj.xtc"/>
+            <output name="output1" format="dcd">
```
Comment (simonbray) [inline lib/galaxy/datatypes/converters/xtc_or_trr_to_dcd.xml:18]: The dcd files have a time stamp in the binary which is a bit annoying. I don't know of a better way to compare than the file size.

**Review by mvdbeek — COMMENTED** [review]
Can you set the tool profile version to something recent (20.09 ?) and PR the test data to https://github.com/galaxyproject/galaxy-test-data ? Otherwise this looks good to me.

**Comment by mvdbeek** [comment]
If you can drop the test files from the PR I'd merge the PR.

**Comment by simonbray** [comment]
> If you can drop the test files from the PR I'd merge the PR.

Ah, it seems I misunderstood your last comment. I've removed them now.

**Comment by mvdbeek** [comment]
My bad, but looks good now! Just waiting for the tests to pass.

**Review by mvdbeek — APPROVED** [review]

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Thanks, Simon!

**Pull request merged**